### ISSUE TITLE
Add WiFi credentials on public places with WiFi

### ISSUE THREAD

**Issue by alvinsartor** [mapsme/omim#11052]
The possibility to add WiFi credentials, rather than only signalling if a place has WiFi or not, would be extremely useful for the users and would definitely boost the app usage (while abroad, but also while in the home country).

From the UI point of view the changes would be in line with the current style and quickly assimilable by the users.
It wouldn't weight too much on the DB (1 more table, with a couple of fields + the key).

![Screenshot_20190615-180024_Word](https://user-images.githubusercontent.com/21270278/59550673-9d18e980-8f97-11e9-9a14-8223a3e90990.jpg)

When editing a place, under the WiFi toggle some inputbox would allow to insert WiFi name(s) and password(s).

![Screenshot_20190615-173357_Word](https://user-images.githubusercontent.com/21270278/59550671-94281800-8f97-11e9-8b6e-1b16aa232d4b.jpg)

The WiFi field would become clickable and show all the passwords.

The community would keep the passwords updated, like the rest of the info in the app.

**Comment by thany** [comment]
Should be a feature request for OSM.org first. But I doubt that it will go through 🤔

**Comment by pataquets** [comment]
I was thinking on a possible approach, which could be pulling such info from some third party service. Googling for "public WiFi password database" lists wifimap.io, but I could not find any API or service for querying it. If anyone knows some free/open database, it might be useful for pulling data beforehand to have it available when offline. Just my 2c.

**Comment by thany** [comment]
@pataquets I'd agree with pulling an offline database into the app, but I'd be against using a third-party API, because MAPS.ME has been becoming far too "online" in the past years already. I want it to be truly offline, so a downloadable database would be best.

However, since the team behind MAPS.ME has gone commercial (rather, even more so than it already had gone), I doubt they'll do anything without it being beneficial to their revenue model.

**Comment by pataquets** [comment]
Yes, I fully agree. I meant to download it beforehand as you can do with maps, maybe selecting a region or box. Ideally, it would work both online and offline.

I started using Maps.me mainly because its offline features, which is a must for me.
